### PR TITLE
T27795 (Debian)

### PR DESCRIPTION
**Pull request by GeorgesStavracas**
https://phabricator.endlessm.com/T27795

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -70,6 +70,7 @@ Vcs-Browser: https://salsa.debian.org/gnome-team/gnome-shell
 Package: gnome-shell
 Architecture: linux-any
 Depends: adwaita-icon-theme,
+         dconf-cli,
```
Comment (ptomato) [inline debian/control]: Can this be pushed up to Debian, presumably that's where the postinst script comes from?

File: debian/control [hunk 2]
```diff
@@ -14,6 +14,11 @@ Build-Depends: asciidoc-base,
                dh-sequence-gnome,
                eos-metrics-0-dev,
                gir1.2-accountsservice-1.0 <!nocheck>,
+               gir1.2-animations-dbus-0 <!nocheck>,
+               gir1.2-animation-glib-0 <!nocheck>,
+               gir1.2-eosmetrics-0 <!nocheck>,
+               gir1.2-eos-parental-controls-0 <!nocheck>,
```
Comment (ptomato) [inline debian/control]: This should be `gir1.2-malcontent-0`, I think? (also below)

**Comment by ptomato** [comment]
There's also a bash completion file that should be accounted for:
```
dh_missing: usr/share/bash-completion/completions/gnome-extensions exists in debian/tmp but is not installed to anywhere
	The following debhelper tools have reported what they installed (with files per package)
	 * dh_install: gnome-shell (7), gnome-shell-common (8)
	 * dh_installdocs: gnome-shell (2), gnome-shell-common (0)
	 * dh_installman: gnome-shell (0), gnome-shell-common (0)
	If the missing files are installed by another tool, please file a bug against it.
	When filing the report, if the tool is not part of debhelper itself, please reference the
	"Logging helpers and dh_missing" section from the "PROGRAMMING" guide for debhelper (10.6.3+).
	  (in the debhelper package: /usr/share/doc/debhelper/PROGRAMMING.gz)
	Be sure to test with dpkg-buildpackage -A/-B as the results may vary when only a subset is built
	For a short-term work-around: Add the files to debian/not-installed
dh_missing: missing files, aborting
```

File: debian/control (hunk 2 above)
Comment (GeorgesStavracas) [reply debian/control]: Yes

File: debian/control (hunk 1 above)
Comment (ptomato) [reply debian/control]: I guess you removed this instead?

Comment (GeorgesStavracas) [reply debian/control]: No, the commit is still in the branch.

Comment (ptomato) [reply debian/control]: I see, it was in a collapsed section.

**Pull request closed without merging**